feature.env

```diff
@@ -13,3 +13,5 @@ API_CLIENT_ID=clientid
 API_CLIENT_SECRET=clientsecret
 SYSTEM_CLIENT_ID=clientid
 SYSTEM_CLIENT_SECRET=clientsecret
+
+EDIT_GOALS_ENABLED=true
```

Comment:
As devs we'll also need to add the same `EDIT_GOALS_ENABLED=true` to our local `.env` file 👍